c2corg_ui/__init__.py

```diff
@@ -41,6 +42,23 @@ def main(global_config, **settings):
     # view for assets with cache buster
     _add_static_view(config, 'static', 'c2corg_ui:static')
 
+    # favicon stuff
+    filenames = [
+        "android-chrome-192x192.png",
+        "android-chrome-384x384.png",
+        "apple-touch-icon.png",
+        "browserconfig.xml",
+        "favicon-16x16.png",
+        "favicon-32x32.png",
+        "favicon.ico",
+        "manifest.json",
+        "mstile-150x150.png",
+        "safari-pinned-tab.svg"
+    ]
+    config.add_asset_views('c2corg_ui:static/favicons',
+                           filenames=filenames,
+                           http_cache=60*60*24*365)
```

Comment:
As this is a fixed URL (ie: it doesn't have an identifier in the path name like css and js files) I'd go for sth like 1 day or 1 week, in case we want to change the icon at some point.